packages/Sandblocks-Smalltalk/SBStRegex.class.st

```diff
@@ -84,7 +90,7 @@ SBStRegex >> createExample: aString [
 { #category : #'as yet unclassified' }
 SBStRegex >> currentTextMorph [
 
-	^ text
+	^ text currentTextMorph
```

Comment:
DANGER! This fixes the completion action, but breaks the corresponding undo in a way that will hang up your image uninterruptably! Shall we revert this for now or do you have an idea for a simple fix?

Reply:
the StRegex was pretty broken in general, I updated its code to use the newer Sandblocks APIs, afterwards I didn't notice any more odd behavior https://github.com/hpi-swa/sandblocks/commit/3ff4fa4dbdb953c800b977d2611f561002b5dd68